libmscore/box.h

```diff
@@ -153,6 +153,8 @@ class VBox : public Box {
       QVariant getProperty(Pid propertyId) const override;
       void layout() override;
 
+      virtual void startEditDrag(EditData&) override;
```

Comment:
just curious, is there any reason to mark it virtual here?

Reply:
No, not really. It was just copied from `Box`, line 54. Should I remove it?

Reply:
Yeah, it's better to remove it, because it'll ported in master soon

Reply:
Done!

Reply:
cool, thanks